src/EnergyPlus/ScheduleManager.cc

```diff
@@ -283,7 +284,8 @@ namespace Sched {
         s_sched->daySchedules.push_back(daySched);
         s_sched->dayScheduleMap.insert_or_assign(std::move(Util::makeUPPER(daySched->Name)), daySched->Num);
 
-        daySched->tsVals.resize(Constant::iHoursInDay * s_glob->TimeStepsInHour);
+        // When InitConstantScheduleData is called, TimeStepsInHour is 0, so we ensure 24
+        daySched->tsVals.assign(Constant::iHoursInDay * max(1, s_glob->TimeStepsInHour), 0.0);
```

Comment:
Perhaps I should have kept resize only, but the assign is clearer and initializes.